src/components/molecules/Settings/Account/PasswordModal/index.tsx

```diff
@@ -30,32 +31,97 @@ const PasswordModal: React.FC<Props> = ({ isVisible, onClose, hasPassword, updat
   const theme = useTheme();
 
   const [password, setPassword] = useState("");
+  const [regexMessage, setRegexMessage] = useState("");
   const [passwordConfirmation, setPasswordConfirmation] = useState("");
   const [disabled, setDisabled] = useState(true);
 
+  const tooShortRegex = /^(?=.{1,7}$)/;
+  const tooLongRegex = /^(?=.{25,}$)/;
+  const whitespaceRegex = /(?=.*\s)/;
+  const highSecurityRegex = /^(?=.*[a-z])(?=.*[A-Z])((?=(.*\d){2}))/;
+  const medSecurityRegex = /^((?=.*[a-z])(?=.*[A-Z])|(?=.*[A-Z])(?=.*\d)|(?=.*[a-z])(?=.*\d))/;
+  const lowSecurityRegex = /^((?=\d)|(?=[a-z])|(?=[A-Z]))/;
```

Comment:
They are static values so they can be moved out of the component function, otherwise each regex will be re-created every render time